### ISSUE TITLE
A11y_Model Builder_Recommendation_Data_Contrast(Minimum): Color contrast ratio between the text and the background is 3.949:1 which is less than 4.5:1

### ISSUE THREAD

**Issue by Samhitha66** [dotnet/machinelearning-modelbuilder#2687]
### GitHubTags

#A11ySev3; #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Win11; #Visual Studio Client; #SH_ModelBuilder_Win32_May2023; #Contrast(Minimum); #Win32; #Benchmark; #WCAG1.4.3;

### Environment Details
Application Name: Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.1
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621

### Repro Steps

1. Launch the Visual Studio with at least one of the .NET workload.
2. Tab till enable ML.NET Model Builder (Tools -> Options -> Environment -> Preview Features) and select it.
3. Create a new .NET project or open an existing .NET project.
4. Open context menu(Right-click) of the project in the solution explorer.
5. Tab till Add and then tab till Machine Learning Model and press enter.
6. Model Builder dialog will open. Tab till Scenario tab.
7. Select Scenario screen will appear.
8. Tab till "Recommendation" and press enter.
9. Setup environment screen will appear, Tab till Local CPU tile and select it.
10. Tab till Next button and press enter.
11. Data screen will appear.
12. Tab till Path after connecting to any server.
13. Turn on Color contrast analyzer.
14. Verify whether the color contrast ratio between the text and the background is 3.949:1 which is less than 4.5:1 or not.

### Actual Result
Color contrast ratio between the text and the background is 3.949:1 which is less than 4.5:1

### Expected Result
Color contrast ratio between the text and the background should be greater than or equals to 4.5:1

### User Impact
Low vision user will not be able to get the information regarding the content.

### Attachment
![Screenshot (841)](https://github.com/dotnet/machinelearning-modelbuilder/assets/99243079/b5507d30-75f0-4706-81b5-f996c9c2627a)

**Comment by zewditu** [comment]
@Samhitha66, ready to validate by Model builder version 16.17.0.2332602.

**Comment by Samhitha66** [comment]
Regressed the issue in the below environment
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.4
The issue is fixed. Hence closing the bug.
Please find the attachment for the reference.
#Regressed:06-30-23; #Closed;
![image](https://github.com/dotnet/machinelearning-modelbuilder/assets/99243079/02af5103-3be6-4e51-b65e-8fc0e08b4260)
